README.md

```diff
@@ -7,31 +7,14 @@ See the [Documentation](https://matrix-org.github.io/matrix-authentication-servi
 
 ## Running
 
-- [Install Rust and Cargo](https://www.rust-lang.org/learn/get-started)
-- [Install Node.js and npm](https://nodejs.org/)
-- [Install Open Policy Agent](https://www.openpolicyagent.org/docs/latest/#1-download-opa)
-- Clone this repository
-- Build the frontend
-  ```sh
-  cd frontend
-  npm ci
-  npm run build
-  cd ..
-  ```
-- Build the Open Policy Agent policies
-  ```sh
-  cd policies
-  make
-  # OR, if you don't have `opa` installed and want to build through the OPA docker image
-  make DOCKER=1
-  cd ..
-  ```
-- Generate the sample config via `cargo run -- config generate > config.yaml`
-- Run a PostgreSQL database locally
-  ```sh
-  docker run -p 5432:5432 -e 'POSTGRES_USER=postgres' -e 'POSTGRES_PASSWORD=postgres' -e 'POSTGRES_DATABASE=postgres' postgres
-  ```
-- Update the database URI in `config.yaml` to `postgresql://postgres:postgres@localhost/postgres`
-- Run the database migrations via `cargo run -- database migrate`
-- Run the server via `cargo run -- server -c config.yaml`
+- [Observe and install requirements](https://matrix-org.github.io/matrix-authentication-service/usage/installation.html#requirements)
+- [Install Matrix Authentication Service](https://matrix-org.github.io/matrix-authentication-service/usage/installation.html#installing-from-the-source)
+- [Generate the sample config](https://matrix-org.github.io/matrix-authentication-service/usage/configuration.html)
+- [Provide the database](https://matrix-org.github.io/matrix-authentication-service/usage/installation.html#database)
+- [Customize your minimal configuration](https://matrix-org.github.io/matrix-authentication-service/usage/configuration.html#minimal-configuration)
+  - `database.uri`
+- [Run the database migrations](https://matrix-org.github.io/matrix-authentication-service/usage/usage.html#running)
+- [Run the server](https://matrix-org.github.io/matrix-authentication-service/usage/usage.html#running)
 - Go to <http://localhost:8080/>
+
+- Or use the [docker image](https://matrix-org.github.io/matrix-authentication-service/usage/installation.html#running-from-the-docker-image) alternatively.
```

Comment:
This whole section is very weird to me. I think this should stay a quick "how to get up and running" for deployment, and have dedicated documentation on how to run in production. Can you revert that change for now?

Reply:
I will revert it.

My concern was that there are two instructions in different places and the content is not the same and also both need to be maintained.